src/helm/benchmark/adaptation/adapters/test_language_modeling_adapter.py

```diff
@@ -54,3 +58,116 @@ def test_fits_tokens_within_context_window(self):
 
         # Ensure the pred_tokens are correct
         assert pred_tokens == [TokenizationToken(464)] + [TokenizationToken(262)] * 2047
+
+    def test_prompt_truncated(self):
+        # ============= 1. Test that the prompt is trucanted correctly when it is too long =============
```

Comment:
Don't need all the "============= 1." headers; and "Step 1.1" headers; you'll have to modify all of them if you add / remove a step.